ra2ce/graph/network_config_wrapper.py

```diff
@@ -103,14 +97,18 @@ def _get_file_entry(expected_file: Path) -> Optional[Path]:
                 logging.info(f"Existing graph/network found: {expected_file}.")
             return _value
 
-        return {
-            _ep.stem: _get_file_entry(_ep)
-            for _ep in map(lambda x: output_graph_dir / x, _network_filenames)
-        }
+        return GraphFiles(
+            **{
+                _ep.stem: _get_file_entry(_ep)
+                for _ep in map(
+                    lambda x: output_graph_dir.joinpath(x), _network_filenames
+                )
+            }
+        )
```

Comment:
Is there an easier way to do this?

Reply:
You could make this a classmethod in the `GraphFiles` and then set the values 1-to-1 as you know what you are exactly mapping.